libgc/aix_irix_threads.c

```diff
@@ -116,6 +118,15 @@ pthread_mutex_t GC_suspend_lock = PTHREAD_MUTEX_INITIALIZER;
 pthread_cond_t GC_suspend_ack_cv = PTHREAD_COND_INITIALIZER;
 pthread_cond_t GC_continue_cv = PTHREAD_COND_INITIALIZER;
 
+#if HAS_PTHREAD_PRIO_INHERIT
+__attribute__((constructor))
+__attribute__((unused))
+static void init_aix_irix_mutex(void)
+{
+	mono_os_static_mutex_init(&GC_suspend_lock);
```

Comment:
I think we shouldn't change the Boehm code.
And we shouldn't make Boehm depend on a Mono header.